packages/atomic/cypress/integration/facet-selectors.ts

```diff
@@ -21,11 +23,18 @@ export function createBreadcrumbShadowAlias() {
     .as('breadcrumbClearAllFilter');
 }
 
-export function createAliasShadow(field: string) {
-  const facetSelector = `${FacetSelectors.facetStandard}[field="${field}"]`;
+export function createAliasShadow(field: string, facetMainSelector?: string) {
+  facetMainSelector = facetMainSelector
+    ? facetMainSelector
+    : FacetSelectors.facetStandard;
+  const facetSelector = `${facetMainSelector}[field="${field}"]`;
   cy.get(facetSelector).shadow().find('div.facet div').as('facetShadow');
 }
-export function createAliasFacetUL(field: string) {
-  createAliasShadow(field);
-  cy.get('@facetShadow').find('div:nth-child(2) ul').as('facet_ul');
+export function createAliasFacetUL(field: string, facetMainSelector?: string) {
+  createAliasShadow(field, facetMainSelector);
+  cy.get('@facetShadow').find('ul').as('facet_ul');
+  cy.get('@facet_ul').find('li:nth-child(1)').as('firstFacetValue');
```

Comment:
A lot of different files are using these "hardcoded" aliases (notably facet utils). Is there someway to export these aliases as variables so they can easily be imported and used in these different files?

Reply:
yup, I can try that

Reply:
I just uploaded new version with the alias, can you check it again if it makes sense now?